nixos/modules/services/networking/i2pd.nix

```diff
@@ -106,10 +106,6 @@ let
         description = "Number of ElGamal/AES tags to send.";
         default = 40;
       };
-      destination = mkOption {
```

Comment:
I guess we don't need `mkRemovedOptionModule`?

Reply:
tbh, I don't know why it was added in the common in the first place. Looking at the docs it shouldn't be used anyhow by i2pd, so defining it for incoming tunnels should do nothing. So just removing it will result in build error if somebody defined it, but it should be just removed from the user config as well, so should be fine?